src/Razor/test/Microsoft.AspNetCore.Razor.OmniSharpPlugin.Test/MSBuildProjectManagerTest.cs

```diff
@@ -207,12 +206,15 @@ public async Task ProjectLoadedAsync_AddsNewProjectWithDocument()
             var projectManager = CreateProjectSnapshotManager();
             msbuildProjectManager.Initialize(projectManager);
             var args = new ProjectLoadedEventArgs(
-                ProjectId.CreateNewId(),
-                projectInstance,
-                diagnostics: Enumerable.Empty<MSBuildDiagnostic>().ToImmutableArray(),
+                id: null,
+                project: null,
+                Guid.NewGuid(),
```

Comment:
Could we provide the parameter name for this one? I ASSUME it's the ProjectId based on the edit.

Reply:
Sure can!